tests/python/gaia-ui-tests/gaiatest/gaia_graphics_test.py

```diff
@@ -130,6 +142,12 @@ def image_compare(self, target_img, ref_img, diff_img, fuzz_value):
             err = err.replace('\n', '')
             message = 'WARNING: ' + err + ' pixels mismatched between ' + target_img + ' and ' + ref_img + '\n'
 
+            # move the diff image and the target image to a separate folder
+            if not os.path.exists(self.mismatch_path):
+                os.makedirs(self.mismatch_path)
+            os.rename(target_img, os.path.join(self.mismatch_path, target_img[target_img.rfind("/")+1:]))
```

Comment:
It might worth to extract `target_img.rfind("/")+1` in a variable. I'm not sure what it should represent.